### PR TITLE
CB-30152: Release Yara v2.1.1

### PR DESCRIPTION
**Pull request by jcapolino**
This pull request incorporates all of the good work that was put into Yara 2.1.0 as well as fixes some service related issues.  It also produces a el6 and el7 rpm for easy deployment on both CentOS/Red Hat 6 and 7.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,201 @@
...
+
+The dockerfile in the top-level of the repo contains a CentOS 7 environment for running, building, and testing 
+the connector. 
+
+The provided script `docker-build-rpm.sh` will use docker to build the project, and place the RPM(s) in `${PWD}/RPMS`. 
+
+
+##### Command-line Options
+```text
+usage: yaraconnector [-h] --config-file CONFIG_FILE [--log-file LOG_FILE]
```
Comment (dseidel-b9) [inline README.md]: Has this been renamed to cb-yara-connector? Or is this just gone?

Comment (dseidel-b9) [reply README.md]: I ran `docker-build-rpm.sh` and it seemed to work very smoothly, producing an el7 RPM. How are el6 RPMs created?

File: dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM centos:7
+RUN yum -y install rpm-build epel-release
+RUN yum -y install python36 python36-devel git make gcc gcc-devel automak libtool make
```
Comment (dseidel-b9) [inline dockerfile]: I believe `automak` should be `automake` -- but do we really need this, or `gcc` or `libtool`? Done't see anywhere we're using this stuff.

Comment (dseidel-b9) [inline dockerfile]: I believe `automak` should be `automake` -- but do we really need this, or `gcc` or `libtool`? Done't see anywhere we're using this stuff.

Comment (zacharyestep) [reply dockerfile]: dave's on the money

File: README.md (hunk 1 above)
Comment (jcapolino) [reply README.md]: The executable is named `yaraconnector`.  That is the updated command-line options.

**Pull request merged**